auth-web/src/components/auth/manage-business/AddBusinessDialog.vue

```diff
@@ -69,11 +69,28 @@
               />
             </v-expand-transition>
 
+            <!-- Certified By Name -->
+            <v-expand-transition>
+              <v-text-field
+                v-if="isBusinessIdentifierValid && showCertifier"
+                filled
+                persistent-hint
+                :label="certifiedByNameLabel"
+                :rules="certifiedByNameRules"
+                :maxlength="passcodeMaxLength"
```

Comment:
This doesn't look like the right max length.

What is the max length for the Certified By Name? Or what is a reasonable max number?

Reply:
I am using the same max length for passcode if it is for firm. 150 is a pretty good max number.

Reply:
You're overloading the use of passcodeMaxLength. This is an anti-pattern!

Did you confirm that 150 is a good number with an auth API developer (who can check the db field size) and the UI designers (who can tell you what makes sense from a business point of view)?

Reply:
Sure. I can ask the UI designers.

Reply:
I have just verified with Ethan that 150 is the max length for the certifier name.